Web/RotaMe.Web/Areas/Owner/Views/Projects/List.cshtml

```diff
@@ -0,0 +1,106 @@
+﻿@model IEnumerable<RotaMe.Web.ViewModels.Owner.Projects.ProjectListViewModel>
+
+@{
+    ViewData["Title"] = "List";
+    Layout = "~/Views/Shared/_Layout.cshtml";
+    var projectsCount = 0;
+}
+
+@section Styles{
+    <link rel="stylesheet" href="~/assets/css/plugins/dataTables.bootstrap4.min.css">
+    <link rel="stylesheet" href="~/assets/css/plugins/bootstrap-tagsinput.css">
+}
+
+<div class="pcoded-main-container">
+    <div class="pcoded-content">
+        <partial name="~/Views/Shared/_BreadcrumbPartial.cshtml" model='new BreadcrumbPartialViewModel() { Controller = "Projects", Action = "My Projects List"}' />
+        <!-- [ Main Content ] start -->
+        <div class="row">
+            <!-- liveline-section start -->
+            <div class="col-sm-12">
+                <div class="card">
+                    <div class="card-body">
+                        <ul class="nav nav-tabs mb-3" id="myTab" role="tablist">
+                            <li class="nav-item">
+                                <a class="nav-link active text-uppercase" id="home-tab" data-toggle="tab" href="#home" role="tab" aria-controls="home" aria-selected="true">Projects</a>
+                            </li>
+                        </ul>
+                        <div class="tab-content" id="myTabContent">
+                            <div class="tab-pane fade show active" id="home" role="tabpanel" aria-labelledby="home-tab">
+                                <div class="table-responsive">
+                                    <table id="report-table" class="table table-striped table-hover">
+                                        <thead>
+                                            <tr>
+                                                <th>#</th>
+                                                <th>Project</th>
+                                                <th>Description</th>
+                                                <th>Users in project</th>
+                                            </tr>
+                                        </thead>
+                                        <tbody>
+                                            @foreach (var project in Model)
+                                            {
+                                                projectsCount++;
+                                                var evenOdd = "";
+
+                                                if (projectsCount % 2 == 0)
+                                                {
+                                                    evenOdd = "even";
+                                                }
+                                                else
+                                                {
+                                                    evenOdd = "odd";
+                                                }
```

Comment:
the views are meant to be logicless, can you add that to the view builder rather than being here